diff --git a/client/Makefile b/client/Makefile
index bd65355..4c89a97 100644
--- a/client/Makefile
+++ b/client/Makefile
@@ -24,8 +24,8 @@ client_shared.o : client_shared.c client_shared.h
 
 install : all
 	$(INSTALL) -d ${DESTDIR}$(prefix)/bin
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} mosquitto_pub ${DESTDIR}${prefix}/bin/mosquitto_pub
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} mosquitto_sub ${DESTDIR}${prefix}/bin/mosquitto_sub
+	$(INSTALL)  mosquitto_pub ${DESTDIR}${prefix}/bin/mosquitto_pub
+	$(INSTALL)  mosquitto_sub ${DESTDIR}${prefix}/bin/mosquitto_sub
 
 uninstall :
 	-rm -f ${DESTDIR}${prefix}/bin/mosquitto_pub
diff --git a/config.mk b/config.mk
index a869646..7dd6f5f 100644
--- a/config.mk
+++ b/config.mk
@@ -241,7 +241,7 @@ ifeq ($(WITH_DOCS),yes)
 endif
 
 INSTALL?=install
-prefix=/usr/local
+prefix=/usr
 mandir=${prefix}/share/man
 localedir=${prefix}/share/locale
 STRIP?=strip
diff --git a/lib/Makefile b/lib/Makefile
index 825fcea..9b7c05c 100644
--- a/lib/Makefile
+++ b/lib/Makefile
@@ -25,7 +25,7 @@ all : libmosquitto.so.${SOVERSION} libmosquitto.a
 
 install : all
 	$(INSTALL) -d ${DESTDIR}$(prefix)/lib${LIB_SUFFIX}/
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} libmosquitto.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquitto.so.${SOVERSION}
+	$(INSTALL) libmosquitto.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquitto.so.${SOVERSION}
 	ln -sf libmosquitto.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquitto.so
 	$(INSTALL) -d ${DESTDIR}${prefix}/include/
 	$(INSTALL) mosquitto.h ${DESTDIR}${prefix}/include/mosquitto.h
diff --git a/lib/cpp/Makefile b/lib/cpp/Makefile
index 8b627d3..cdb2923 100644
--- a/lib/cpp/Makefile
+++ b/lib/cpp/Makefile
@@ -10,7 +10,7 @@ all : libmosquittopp.so.${SOVERSION}
 
 install : all
 	$(INSTALL) -d ${DESTDIR}$(prefix)/lib${LIB_SUFFIX}/
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} libmosquittopp.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquittopp.so.${SOVERSION}
+	$(INSTALL) libmosquittopp.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquittopp.so.${SOVERSION}
 	ln -sf libmosquittopp.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquittopp.so
 	$(INSTALL) -d ${DESTDIR}${prefix}/include/
 	$(INSTALL) mosquittopp.h ${DESTDIR}${prefix}/include/mosquittopp.h
diff --git a/src/Makefile b/src/Makefile
index 2cfb7d4..9a97644 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -103,10 +103,10 @@ mosquitto_passwd.o : mosquitto_passwd.c
 
 install : all
 	$(INSTALL) -d ${DESTDIR}$(prefix)/sbin
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} mosquitto ${DESTDIR}${prefix}/sbin/mosquitto
+	$(INSTALL) mosquitto ${DESTDIR}${prefix}/sbin/mosquitto
 	$(INSTALL) mosquitto_plugin.h ${DESTDIR}${prefix}/include/mosquitto_plugin.h
 ifeq ($(WITH_TLS),yes)
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} mosquitto_passwd ${DESTDIR}${prefix}/bin/mosquitto_passwd
+	$(INSTALL) mosquitto_passwd ${DESTDIR}${prefix}/bin/mosquitto_passwd
 endif
 
 uninstall :
